pokemongo_bot/cell_workers/initial_transfer_worker.py

```diff
@@ -14,10 +14,18 @@ def work(self):
         ignlist = self.config.ign_init_trans.split(',')
 
         if self.config.cp:
-            logger.log('[x] Will NOT transfer anything above CP {} or these {}'.format(
-                self.config.cp, ignlist))
+            if self.config.pokemon_potential:
+                logger.log('[x] Will NOT transfer anything above CP {}, IV {} or these {}'.format(
+                    self.config.cp, self.config.pokemon_potential, ignlist))
+            else:
+                logger.log('[x] Will NOT transfer anything above CP {} or these {}'.format(
+                    self.config.cp, ignlist))
         else:
-            logger.log('[x] Preparing to transfer all Pokemon duplicates, keeping the highest CP of each one type.')
+            if self.config.pokemon_potential:
+                logger.log('[x] Will NOT transfer anything above IV {} or these {}'.format(
+                    self.config.pokemon_potential, ignlist))
+            else:
+                logger.log('[x] Preparing to transfer all Pokemon duplicates, keeping the highest CP of each one type.')
```

Comment:
Could you change lines 16 -> 28 to:

```
        if not self.config.pokemon_potential and not self.config.cp:
            logger.log('[x] Preparing to transfer all Pokemon duplicates, keeping the highest CP of each one type.')
        else:
            logger.log('[x] Will NOT transfer anything above CP {}, IV {} or these {}'.format(
                self.config.cp, self.config.pokemon_potential, ignlist)) 
```

That way we don't have 3 branches of this code doing the same thing :)